main.go

```diff
@@ -294,6 +295,9 @@ func main() {
 		os.Exit(1)
 	}
 
+	log.Infof("inserting rules")
+	network.InsertRules(network.RulesToInsert(flannelFwdChain))
```

Comment:
This only works when the FLANNEL-FORWARD chain  is already created once.  So on a clean system, this won't work.
Better to move the code up from below right above the InsertRules call
 if err = network.AddNewChain("filter", flannelFwdChain); err != nil {                                                                                                                                                                                                  //fall back to FORWARD chain                                                                                                                                                                                                                                         flannelFwdChain = "FORWARD"                                                                                                                                                                                                                                        }